src/modules/itemsCounter.js

```diff
@@ -0,0 +1,13 @@
+const itemsCounter = () => {
+  const numberItemsElement = document.querySelector('.number-items');
+  const results = document.querySelectorAll('.character-card');
+  let cont = 0;
+
+  results.forEach(() => {
+    cont += 1;
+  });
+
+  numberItemsElement.textContent = ` (${cont})`;
+};
```

Comment:
Good job on implementing the counter feature. However you aren't covering the edge case of when there are no items at all in the Dom. Please consider this in order to make the code more robust.